src/config.ts

```diff
@@ -32,15 +32,42 @@ export interface Config {
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  imports: ['k8s'],
 };
 
 export function readConfigSync(): Config {
   let config: Config = DEFAULTS;
   if (fs.existsSync(CONFIG_FILE)) {
+    const parsedYaml: Config = yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8'));
     config = {
       ...config,
-      ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
+      ...parsedYaml,
     };
   }
   return config;
+}
+
+export async function addImportToConfig(source: string) {
+  let curConfig = readConfigSync();
+
+  if (!curConfig.imports?.includes(source)) {
```

Comment:
```suggestion
  const currImports = curConfig.imports ?? [];
  if (!currImports.includes(source)) {
```

And then push directory to `currImports` below